src/netstandard/ref/System.Buffers.cs

```diff
@@ -120,6 +134,51 @@ public abstract partial class ReadOnlySequenceSegment<T>
         public long RunningIndex { get { throw null; } protected set { } }
     }
     public delegate void ReadOnlySpanAction<T, in TArg>(System.ReadOnlySpan<T> span, TArg arg);
+    public ref partial struct SequenceReader<T> where T : struct, System.IEquatable<T>
+    {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public SequenceReader(System.Buffers.ReadOnlySequence<T> sequence) { throw null; }
+        public long Consumed { get { throw null; } }
+        public System.ReadOnlySpan<T> CurrentSpan { get { throw null; } }
+        public int CurrentSpanIndex { get { throw null; } }
+        public bool End { get { throw null; } }
```

Comment:
In .NET Core a bunch of these members are marked as `readonly`, e.g. End, Sequence, Position, etc.  I'm not seeing that here.

Reply:
That's odd. I'll investigated why that is.

Reply:
The problem is that the tool I use to create the assembly is using an outdated version of CCI extensions (because BuildTools is dead and the current version is from Arcade). That's mildly concerning because that might mean that we've missed other cases like this. I'm updating the tools and ill issue a diff.

Reply:
Alright, I've updated the PR and at least fixed the readonly annotations for this part. I'll go through the rest after this is merged.

Reply:
@stephentoub is that enough for you to sign-off?

Reply:
BTW, I finished the diff and the only other types that miss read-only modifiers are in numerics. I have submitted #1508 to address that.